js/clinicalTimeline.js

```diff
@@ -261,6 +284,121 @@ window.clinicalTimeline = (function(){
+      .domain([minDayTick, maxDayTick])
+      .range([0 , overviewAxisWidth]);
+
+    var xScaleOverviewZoomed = d3.time.scale()
+      .domain([0, -zoomedWidth])
+      .range([0 , overviewAxisWidth]);
+
+    var xScaleRectangle = d3.time.scale()
+      .domain([0, overviewAxisWidth])
+      .range([0 , -zoomedWidth]);
+
+    var overviewAxis = d3.svg.axis().scale(xScaleOverview).orient("bottom")
+      .tickFormat(function(d) { 
+        return formatTime(daysToTimeObject(d.valueOf()), "months");
+      })
+      .ticks(tickTemp.length)
+      .tickSize(3)
+      .tickPadding(4);
+
+     var overviewAxisDuplicate = d3.svg.axis().scale(xScaleOverview).orient("top")
+      .tickFormat(function(d) { 
+        return formatTime(daysToTimeObject(d.valueOf()), "months");
+      })
+      .ticks(tickTemp.length)
+      .tickSize(3)
+      .tickPadding(0);
+
+    var rectangleOverviewWidth = width/zoomFactor;
+
+    overview.append("g")
+      .attr("class", "x axis overviewaxis")
+      .attr("transform", "translate(0,25)")
+      .call(overviewAxis);
+
+    overview.append("g")
+      .attr("class", "x axis overviewaxis duplicate")
+      .attr("transform", "translate(0,44)")
+      .call(overviewAxisDuplicate);
+
+    var dragChart = d3.behavior.drag()
+      .on("drag", function(d,i) {
+        if(chart.scrolledX()){
+          var zoomedBeginTick = xScaleOverviewZoomed(chart.scrolledX()); 
+          d3.select(".pos_rectangle").attr("x", zoomedBeginTick);
+          overviewX = zoomedBeginTick;
+        }
+      });
+
+    var dragRectangle = d3.behavior.drag()
+      .on("drag", function(d,i) {
+        var x  = parseInt(d3.select(".pos_rectangle").attr("x"))+d3.event.dx;
+        if(x > 0 && x < overviewAxisWidth - rectangleOverviewWidth){
+          d3.select(divId+" svg g").attr("transform","translate("+xScaleRectangle(x)+",0)");
+          d3.select(".pos_rectangle").attr("x", x); 
+          overviewX = x;
+        }
+      });
+
+    d3.select(".overview").append("rect")
+      .attr("height", 3)
+      .attr("width", overviewAxisWidth)
+      .attr("x", 0)
+      .attr("y", 22)
+      .attr("fill", "#ccc");
+
+    d3.select(".overview").append("rect")
+      .attr("height", 3)
+      .attr("width", overviewAxisWidth)
+      .attr("x", 0)
+      .attr("y", 44)
+      .attr("fill", "#ccc");
+
+    d3.select(".overview").append("rect")
+      .attr("height", 18)
+      .attr("width", 3)
+      .attr("x", 0)
+      .attr("y", 26)
+      .attr("fill", "#ccc");
+
+    d3.select(".overview").append("rect")
+      .attr("height", 18)
+      .attr("width", 3)
+      .attr("x", overviewAxisWidth-3)
+      .attr("y", 26)
+      .attr("fill", "#ccc");
+
+      var rectangle = d3.select(".overview").append("rect")
+        .attr("height", 16)
+        .attr("width", rectangleOverviewWidth)
+        .attr("x", overviewX)
+        .attr("y", 27)
+        .attr("fill", "rgba(25,116,255, 0.4)")
+        .attr("class", "pos_rectangle")
```

Comment:
In the google style guide they recommend to use hyphens instead of underscores: https://google.github.io/styleguide/htmlcssguide.xml#CSS_Style_Rules